WordPress/src/main/java/org/wordpress/android/ui/prefs/notifications/NotificationsSettingsFragment.java

```diff
@@ -140,8 +139,8 @@ private void removeSightAndSoundsForAPI26() {
 
             PreferenceCategory categorySightsAndSounds =
                     (PreferenceCategory) preferenceScreen.findPreference(getActivity()
-                    .getString(
-                            R.string.pref_notification_sights_sounds));
+                            .getString(
+                                    R.string.pref_notification_sights_sounds));
```

Comment:
Indentation got a bit weird here. Wouldn't it be a bit more readable like this?

```
PreferenceCategory categorySightsAndSounds = (PreferenceCategory) preferenceScreen
        .findPreference(getActivity().getString(R.string.pref_notification_sights_sounds));
```

Reply:
I use autoformatter all the time, and it sometimes messes up the indentation :) Fixed it!